### PR TITLE
docs(guidelines): add date time formats. closes #248

### LINKED ISSUE #248: Date time format
- [x] Find all occurences in Solution Center
- [x] Choose a standard single date time format (English) for all products
- [x] Break format into smaller standalone parts
- [x] Implement in dress code docs

Note: First we'll introduce English only. Later comes i18n.

### PR DESCRIPTION
**Pull request by xonic**
It basically adds a new navigation section on the left side of the demo/docs page called "GUIDELINES" with first entry "Date Time Format".

### CODE REVIEW

File: docs/demo/materials/01-guidelines/01-date-time-format.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+notes: |
+  Dress code is currently based on British local (en-gb).
+---
```
Comment (sambhav-gore) [inline docs/demo/materials/01-guidelines/01-date-time-format.html]: It should be `locale`

**Comment by xonic** [comment]
@sambhav-gore we've been discussing this as well and opinions diverge. The guidelines added here serve as guidelines for designers as well as for people who'd like to integrate their product into Solution Center. If you're part of Solution Center, you should conform to the formats we use. Since we don't have i18n yet, the suggestion is to follow this format based on British locale. 

So, the demo page and documentation serves two purposes: 
- design and integration guidelines
- documentation of framework components

Now, when you start building with dress-code, you're only gonna get the CSS and are free to do whatever you like. If you'd like to integrate with SC, the guidelines will help you conform and ultimately create a more consistent and pleasing experience for the user.

Maybe it would make sense to rename GUIDELINES to SOLUTION CENTER INTEGRATION GUIDELINES or anything similar that is more distinct from the rest?

What do others think?

**Comment by xonic** [comment]
Other examples of such guidelines can be found here:
https://www.gov.uk/service-manual/user-centred-design/resources/patterns/dates.html
https://www.lightningdesignsystem.com/guidelines/localization/
https://material.google.com/patterns/data-formats.html

**Comment by sambhav-gore** [comment]
After discussing this with @xonic I think it is okay to have these guidelines here but we should look at making these guideline more descriptive in future and provide use-case guidelines as well - such as when is it recommended to use which format while not having any direct references to Solution Center or Brand Solutions so that it is generic enough for anyone to use it.

**Comment by sambhav-gore** [comment]
:+1:

**Comment by sambhav-gore** [comment]
👍

**Comment by gabrielhl** [comment]
@xonic You need to update your branch to merge the current develop file structure into it and open a new PR for this. When I merged this one caused duplication of folders which broke Fabricator.

**Pull request merged**